src/tests/sorting/test_sorting.f90

```diff
@@ -14,7 +14,7 @@ program test_sorting
     integer(int32), parameter :: char_size = 26**4
     integer(int32), parameter :: string_size = 26**3
     integer(int32), parameter :: block_size = test_size/6
-    integer, parameter        :: repeat = 8
+    integer, parameter        :: repeat = 1
```

Comment:
Is this change deliberate?

Reply:
FWIW this decreases the number of times the sorts are repeated, making the test complete faster, but  potentially resulting in more noise in the comparison of the different tests.

Reply:
It was to speed up a it the tests. I forgot to revert it. Thank you for mentioning it. I wil revert it.